seed/lib/mcm/cleaners.py

```diff
@@ -33,6 +34,46 @@
 PUNCT_REGEX = re.compile('[{0}]'.format(
     re.escape(string.punctuation.replace('.', '').replace('-', '')))
 )
+# Mapping of specific characters to their normalized versions (need to expand this list)
+CHAR_MAPPING = {
+    ord('“'): '"',
+    ord('”'): '"',
+    ord('‘'): "'",
+    ord('’'): "'",
+    ord('′'): "'",
+    ord('″'): '"',
+    ord('‴'): "'''",
+    ord('…'): '...',
+    ord('•'): '*',
+    ord('⁄'): '/',
+    ord('×'): 'x',
+    ord('⁓'): '~',
+    # mdash, ndash, horizontal bar
+    ord('–'): '-',
+    ord('—'): '--',
+    ord('―'): '-',
+    ord('¬'): '-',
+    # guillemets?
```

Comment:
@kflemin  -- should we normalize on these too?

Reply:
you could normalize them to single and double quotation marks